### PR TITLE
[doxygen] update doxygen tag pattern

### PR DESCRIPTION
**Pull request by niley7464**
nnstreamer's tensor-transform has a CI error.
This patch updates doxygen tag pattern written in upper case.

See also #736 
Signed-off-by: Yelin Jeong <yelini.jeong@samsung.com>

### CODE REVIEW

File: ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh [hunk 1]
```diff
@@ -151,15 +151,8 @@ function pr-prebuild-doxygen-tag(){
...
-			    # @todo it should be able to know if it is inside a comment or not!
-                            if [[ $line =~ [\ \t]*\*[\t \t]*"@"[A-Z] ]]; then
-                                echo "[ERROR] File name: $curr_file, $idx line, The doxygen tag sholud be written in lower case." >> $report_path
-                                check_result="failure"
-                                global_check_result="failure"
-                                latest_failed_file=$curr_file
-                            fi
-                            if [[ $line =~ //[\ \t]*"@"[A-Z] ]]; then
+                            # Let's check a comment statement that begins with '//','/*' or ' *'.
+                            if  [[ ($line =~ "@"[A-Z]) && ($line == "/*"* || $line == "//"* || $line == [[:blank:]]*"*"*)]]; then
```
Comment (myungjoo) [inline ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh:155]: ```==```? not ```=~```?

File: ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh [hunk 2]
```diff
@@ -151,15 +151,8 @@ function pr-prebuild-doxygen-tag(){
...
-			    # @todo it should be able to know if it is inside a comment or not!
-                            if [[ $line =~ [\ \t]*\*[\t \t]*"@"[A-Z] ]]; then
-                                echo "[ERROR] File name: $curr_file, $idx line, The doxygen tag sholud be written in lower case." >> $report_path
-                                check_result="failure"
-                                global_check_result="failure"
-                                latest_failed_file=$curr_file
-                            fi
-                            if [[ $line =~ //[\ \t]*"@"[A-Z] ]]; then
+                            # Let's check a comment statement that begins with '//','/*' or ' *'.
+                            if  [[ ($line =~ "@"[A-Z]) && ($line == "/*"* || $line == "//"* || $line == [[ \t]]*"*"*)]]; then
```
[the remark this answers is not in the document]
Comment (niley7464) [reply ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh]: `[[:blank:]]` is same as `[ \t]`.
( https://en.wikibooks.org/wiki/Regular_Expressions/POSIX_Basic_Regular_Expressions )
But `[[:blank:]]` seems easy to read, so I fixed it to `[[:blank:]]`
Thank you!

File: ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh (hunk 1 above)
Comment (niley7464) [reply ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh:155]: I thought using `=~` can cause a problem when string has a value like this.
```c
str="...*...@DIM...";
```
So I tried to find a string which begins with `//,/*, *`

Comment (myungjoo) [reply ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh:155]: The problem is that you cannot detect when they are indented:
```
$ line="    /* abc @D"
$ line2="/* abc @D"
$ if [[ $line == "/*"* ]]; then echo Y; else echo N; fi
N
$ if [[ $line2 == "/*"* ]]; then echo Y; else echo N; fi
Y
```

Comment (niley7464) [reply ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh]: Thank you for a detailed explanation.
I updated the format to detect indented comment!

**Review by again4you — APPROVED** [review]
In my opinion, updated version looks good.

**Review by myungjoo — APPROVED** [review]
LGTM

**Comment by myungjoo** [comment]
Please deploy the new code to AWS.

**Pull request merged**